app/views/layouts/application.html.erb

```diff
@@ -20,32 +20,23 @@
       only the last one will load correctly in production, as though there is
       a global registry and the last bundle overwrites the entire registry.
     -->
-    <%= javascript_pack_tag 'webpack_bundle' %>
+    <%= javascript_pack_tag 'webpack_bundle', defer: Rails.env.production? %>
     <%= stylesheet_pack_tag 'webpack_bundle' %>
     <!-- Webpack Assets END -->
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload', async: Rails.env.production? %>
 
     <% if user_signed_in? %>
       <!-- Pusher -->
-      <script src="https://js.pusher.com/3.0/pusher.min.js"></script>
+      <script src="https://js.pusher.com/3.0/pusher.min.js" async></script>
```

Comment:
Any reason why `<script>` is being used here instead of `javascript_include_tag`?

Reply:
No reason, I just copied at pasted the script link from these third-party APIs in their README. I can convert this over to javascript_include_tag to be consistent with Rails ERB syntax :)